openwisp_firmware_upgrader/tests/test_models.py

```diff
@@ -12,7 +12,7 @@
 from .. import settings as app_settings
 from ..hardware import FIRMWARE_IMAGE_MAP, REVERSE_FIRMWARE_IMAGE_MAP
 from ..swapper import load_model
-from ..tasks import upgrade_firmware
+from ..tasks import delete_firmware_files, upgrade_firmware
```

Comment:
I am not in favour of the way these tests are written.

Tests for bugs should fail without the fix, that is, if I run these tests on master they should fail (all assertions/subtests), while they should pass with the patch.

However, if you try to run these tests on master they can't be run because `delete_firmware_files` is not yet available. That would be ok if we were adding test for a feature, but we need tests that replicate the bug.